services/user/service/user_service.go

```diff
@@ -67,14 +68,45 @@ func SetUserInfo(id string, user_info models.UserInfo) error {
 	Returns the users associated with the given parameters
 */
 func GetFilteredUserInfo(parameters map[string][]string) (*models.FilteredUsers, error) {
+	// Grab the sortby parameter and delete it to prevent the CreateFilterQuery from using it
+	sort_parameters := parameters["sortby"]
+	delete(parameters, "sortby")
+
 	query, err := database.CreateFilterQuery(parameters, models.UserInfo{})
 
 	if err != nil {
 		return nil, err
 	}
 
 	var filtered_users models.FilteredUsers
-	err = db.FindAll("info", query, &filtered_users.Users)
+
+	if len(sort_parameters) == 1 {
+		// Split by comma to get a slice of sorting parameters
+		// i.e FirstName, LastName --> ["FirstName", "LastName"]
+		sort_parameters = strings.Split(sort_parameters[0], ",")
+
+		// Create and fill the sort fields
+		var sort_fields []database.SortField
+
+		for _, field := range sort_parameters {
+			// Push to lowercase because MongoDB columns are all lowercase
+			field = strings.ToLower(field)
+			field = strings.TrimSpace(field)
+
+			// TODO Add a check here to make sure the requested sort field is actually a field in the model.
```

Comment:
Good idea! We should probably do this for all filter & sort params for other services too.